main.go

```diff
@@ -69,23 +69,29 @@ func main() {
 	var foodParkOptions []foodParkOption = []foodParkOption{}
 
 	foodOptionsDiv.Children().Each(func(i int, s *goquery.Selection) {
-		foodOption := foodParkOption{
-			name: s.Find(viper.GetString("name_filter_query")).Text(),
-		}
+		foodOptionsAnchorTags := s.Find(viper.GetString("anchor_filter_query"))
 
-		url, exists := s.Find("a").Attr("href")
-		if !exists {
-			log.Fatalf("Failed to find URL for food option %s", foodOption.name)
-			os.Exit(1)
-		}
-		if !strings.Contains(url, viper.GetString("target_date")) {
-			log.Fatalf("Failed to find target date string %s in URL %s", viper.GetString("target_date"), url)
-			os.Exit(1)
-		}
+		foodOptionsAnchorTags.Each(func(i int, s1 *goquery.Selection) {
+			url, exists := s1.Attr("href")
 
-		foodOption.url = url
+			if !exists {
+				log.Fatal("Anchor tag does not have a href attribute")
+				os.Exit(1)
+			}
+			if !strings.Contains(url, viper.GetString("target_date")) {
+				log.Fatalf("Failed to find target date string %s in URL %s", viper.GetString("target_date"), url)
+				os.Exit(1)
+			}
 
-		foodParkOptions = append(foodParkOptions, foodOption)
+			name := s1.Parent().Parent().Parent().Prev().Text()
```

Comment:
Would be super cool if we could automate this a bit (I imagine you have similar thoughts), though with HTML trawling being what it is, maybe not 😅 I'll have a think

Reply:
Are you saying that having an extremely fragile assumption that the parents, children, and siblings aren't going to change is a bad idea? Impossible!

Reply:
These kinds of hard-hitting insights are why you're glad I'm looking at this right